gradle/libs.versions.toml

```diff
@@ -12,27 +12,26 @@ kord-cache = "0.5.4" # https://github.com/kordlib/cache
 kotlin-logging = "7.0.0" # https://github.com/oshai/kotlin-logging
 kotlin-logging-old = "3.0.5" # TODO remove after dependency is removed in rest, gateway, voice and core
 slf4j = "2.0.13" # https://www.slf4j.org
-kotlin-node = "20.11.30-pre.765" # https://github.com/JetBrains/kotlin-wrappers
-bignum = "0.3.9" # https://github.com/ionspin/kotlin-multiplatform-bignum
+kotlin-node = "20.14.10-pre.773" # https://github.com/JetBrains/kotlin-wrappers
+bignum = "0.3.10" # https://github.com/ionspin/kotlin-multiplatform-bignum
 stately = "2.0.7" # https://github.com/touchlab/Stately
 fastZlib = "2.0.1" # https://github.com/timotejroiko/fast-zlib
 
 # code generation
-ksp = "2.0.0-1.0.22" # https://github.com/google/ksp
-kotlinpoet = "1.17.0" # https://github.com/square/kotlinpoet
+ksp = "2.0.0-1.0.23" # https://github.com/google/ksp
+kotlinpoet = "1.18.0" # https://github.com/square/kotlinpoet
 
 # tests
 junit-jupiter = "5.10.3" # https://github.com/junit-team/junit5
 junit-platform = "1.10.3"
 mockk = "1.13.11" # https://github.com/mockk/mockk
-bson = "5.1.1" # https://github.com/mongodb/mongo-java-driver
-kbson = "0.5.0" # https://github.com/jershell/kbson
+kbson = "0.4.0" # https://github.com/mongodb/kbson
```

Comment:
jershell/kbson seems to be for BSON (binary), mongodb/kbson for EJSON (text), which seems to be a different thing? https://www.mongodb.com/docs/manual/reference/mongodb-extended-json/

maybe test against both?

Reply:
Since most people will use bson for mongodb purposes I would assume using the mongo one is fine

Reply:
i think i chose jershell/kbson because @NoComment1105 had an issue with that serialization lib https://discord.com/channels/556525343595298817/587324906702766226/1227561614643953675

Reply:
That will happen will all non json encoders, so will happen  with this one too

Reply:
true